### PR TITLE
Iotix: add startup pattern to home.html

### PR DESCRIPTION
**Pull request by jffng**
#### Changes proposed in this Pull Request:

This PR adds a `startup` pattern to Iotix, and includes it home template. 

This is more consistent with other themes, so that users don't have to fiddle with reading settings to get the demo to match their site.

cc @beafialho 

#### To Test:

- Check out this PR
- Go to the site editor, verify home.html appears as expected
- Go to the front, verify front page appears as expected

### CODE REVIEW

File: iotix/patterns/startup.php [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+<!-- /wp:cover -->
+
+<!-- wp:group {"align":"wide","style":{"spacing":{"padding":{"top":"var:preset|spacing|80","right":"var:preset|spacing|50","bottom":"var:preset|spacing|80","left":"var:preset|spacing|50"},"blockGap":"0"}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group alignwide" style="padding-top:var(--wp--preset--spacing--80);padding-right:var(--wp--preset--spacing--50);padding-bottom:var(--wp--preset--spacing--80);padding-left:var(--wp--preset--spacing--50)"><!-- wp:group {"align":"wide","style":{"spacing":{"blockGap":"var:preset|spacing|40"}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group alignwide"><!-- wp:heading {"textAlign":"center"} -->
+<h2 class="wp-block-heading has-text-align-center"><?php echo esc_html__( 'Our Pricing', 'iotix' ); ?></h2>
+<!-- /wp:heading -->
+
+<!-- wp:paragraph {"align":"center"} -->
+<p class="has-text-align-center"><?php echo esc_html__( 'Our pricing structure is built to be affordable for everyone from individual designers to teams.', 'iotix' ); ?></p>
```
Comment (vcanales) [inline iotix/patterns/startup.php]: Could we turn the pricing table into a pattern? Such as what we have on Loudness

Comment (jffng) [reply iotix/patterns/startup.php]: Good idea, done in [d065041](https://github.com/Automattic/themes/pull/7030/commits/d06504129105d8c3ee307f0250d54ed3ad91b9b0)

**Comment by jffng** [comment]
@beafialho would you be able to provide credits for the images used?

**Comment by beafialho** [comment]
@jffng all the 3D illustrations were created by @keoshi.

Here are the credits for the other photos used:
https://stocksnap.io/photo/businessmeeting-people-QVIEE1UZSX
https://stocksnap.io/photo/woman-developers-FVWLQXWZ1P

**Pull request merged**